weave/wandb_interface/wandb_stream_table.py

```diff
@@ -136,57 +63,78 @@ def __init__(
         # technically, we could infer the entity from the API key, but
         # that tends to confuse users.
         if entity_name is None or entity_name == "":
-            raise ValueError(f"Must specify entity_name")
+            raise ValueError("Must specify entity_name")
         elif project_name is None or project_name == "":
-            raise ValueError(f"Must specify project_name")
+            raise ValueError("Must specify project_name")
         elif table_name is None or table_name == "":
-            raise ValueError(f"Must specify table_name")
+            raise ValueError("Must specify table_name")
 
         self._lite_run = InMemoryLazyLiteRun(
             entity_name,
             project_name,
             table_name,
+            config=config,
             group="weave_stream_tables",
-            _hide_in_wb=True,
-            _use_async_file_stream=not _disable_async_file_stream,
+            _hide_in_wb=hidden,
         )
+        if not wandb_public_api().supports_streamtable:
+            raise errors.Error(
+                "Streamtable isn't supported in this version of wandb.  Contact your adminstrator to upgrade."
+            )
```

Comment:
I need to wire this up in wandb server still but it should always be true for SaaS.